src/ports/postgres/modules/convex/mlp.sql_in

```diff
@@ -1346,99 +1349,6 @@ JOIN mlp_regress_prediction USING (id);
 (1 row)
 </pre>
 
-<h4>Regression with Grouping and Mini-Batching</h4>
```

Comment:
Why did we remove this?

Reply:
1- A bit redundant, we already have examples for:
- MLP with grouping
- MLP with minibatching
- minibatching with grouping in the minibatching docs
2- With such a small dataset, it was occasionally failing on my local mac setup.